### PR TITLE
Release notes/ docs for Org API tokens

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/2246

### CODE REVIEW

**Review by chomipi88 — COMMENTED** [review]
see comment

**Comment by jwitz** [comment]
@chomipi88 If you think it's primarily Org Admins that are adding users, I think that's a great idea!

**Comment by chomipi88** [comment]
> 

Adding to an Org... yes definitely only Org Admins.
Maybe we split our user management in Workspaces under Manage Workspaces section?

**Comment by chomipi88** [comment]
Good feedback here on updating the `User Permissions` page to add details about Create, Update, and Delete access to Org and Workspace API Tokens.

https://astronomer.slack.com/archives/C01UA03TPV1/p1684322309476069

File: astro/release-notes.md [hunk 1]
```diff
@@ -21,6 +21,21 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 **Latest CLI Version**: 1.14.1 ([Release notes](cli/release-notes.md))
 
+## May 16, 2023
+
+### Automate Organization management with Organization API tokens
+
+You can now create Organization API tokens to automate key actions across your Organization and all of the Workspaces in it. You can customize the role and expiration date of the token to give it the minimum required permissions for the task it completes. Some common actions that you can automate with Organization API keys are:
```
Comment (jwitz) [inline astro/release-notes.md:28]:
```suggestion
You can now create Organization API tokens to automate key actions across your Organization and all of the Workspaces in it. You can customize the role and expiration date of the token to give it the minimum required permissions for the task it completes. Some common actions that you can automate with Organization API token are:
```

**Comment by chomipi88** [comment]
> Good feedback here on updating the `User Permissions` page to add details about Create, Update, and Delete access to Org and Workspace API Tokens.
> 
> https://astronomer.slack.com/archives/C01UA03TPV1/p1684322309476069

@jwitz Believe this is the only remaining modification for this PR before we merge it.

File: astro/user-permissions.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ An Organization role grants a user some level of access to an Astro Organization
 | Update roles and permissions of existing Organization users      |                         |                                | ✔️                      |
 | Invite a new user to an Organization                             |                         |                                | ✔️                      |
 | Remove a user from an Organization                               |                         |                                | ✔️                      |
+| Create Organization API tokens                                   |                         |                                | ✔️                      |
```
Comment (chomipi88) [inline astro/user-permissions.md:39]:
```suggestion
| Create, Update and Delete Organization API tokens                                   |                         |                                | ✔️                      |
```

File: astro/user-permissions.md [hunk 3]
```diff
@@ -64,5 +65,7 @@ A Workspace role grants a user some level of access to a specific Workspace. The
 | Create, Update and Delete API Keys                  |                      |                      | ✔️                   |
 | Update user roles and permissions                   |                      |                      | ✔️                   |
 | Invite users to a Workspace                         |                      |                      | ✔️                   |
+| Create Workspace API tokens                         |                      |                      | ✔️                   |
```
Comment (chomipi88) [inline astro/user-permissions.md:68]:
```suggestion
| Create, Update and Delete Workspace API tokens                         |                      |                      | ✔️                   |
```

**Review by chomipi88 — APPROVED** [review]
lgtm

**Pull request merged**